src/app/tests/suites/certification/Test_TC_OVENOPSTATE_2_2.yaml

```diff
@@ -0,0 +1,335 @@
+# limitations under the License.
+# Auto-generated scripts for harness use only, please review before automation. The endpoints and cluster names are currently set to default
+
+name: 3.2.2. [TC-OVENOPSTATE-2.2] Start and Stop commands with DUT as Server
+
+PICS:
+    - OVENOPSTATE.S
+
+config:
+    nodeId: 0x12344321
+    cluster: "Basic Information"
+    endpoint: 0
+
+tests:
+    - label: "Note"
+      verification: |
+          This is a simulated example log for instructional purposes only. In real scenarios, the actual log may vary depending on the feature implementation in Reference App.
+      disabled: true
+
+    - label:
+          "Step 1: Commission DUT to TH (can be skipped if done in a preceding
+          test)"
+      verification: |
+          OVENOPSTATE.S.C01 && OVENOPSTATE.S.C02
+      disabled: true
+
+    - label:
+          "Step 2: Manually put the DUT into a state wherein it can receive a
+          Start Command"
+      verification: |
+          Manually put the DUT into a state wherein it can receive a Start Command
+      disabled: true
+
+    - label: "Step 3: TH reads from the DUT the OperationalStateList attribute"
+      PICS: OVENOPSTATE.S.A0003
+      verification: |
+          ./chip-tool operationalstate read operational-state-list 1 1
+
+           Via the TH (chip-tool), verify:
+           - all entries include an ID (enum8) and a label (string)
+           - all provided IDs are in the allowed range
+           - the list includes IDs for Error (0x03), Running (0x01), and Stopped (0x00)
+
+          [1689674049.504261][17222:17224] CHIP:DMG: }
+          [1689674049.504390][17222:17224] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0060 Attribute 0x0000_0003 DataVersion: 2102885775
+          [1689674049.504440][17222:17224] CHIP:TOO:   OperationalStateList: 4 entries
+          [1689674049.504462][17222:17224] CHIP:TOO:     [1]: {
+          [1689674049.504469][17222:17224] CHIP:TOO:       OperationalStateID: 0
+          [1689674049.504476][17222:17224] CHIP:TOO:      }
+          [1689674049.504484][17222:17224] CHIP:TOO:     [2]: {
+          [1689674049.504490][17222:17224] CHIP:TOO:       OperationalStateID: 1
+          [1689674049.504495][17222:17224] CHIP:TOO:      }
+          [1689674049.504503][17222:17224] CHIP:TOO:     [3]: {
+          [1689674049.504508][17222:17224] CHIP:TOO:       OperationalStateID: 2
+          [1689674049.504514][17222:17224] CHIP:TOO:      }
+          [1689674049.504521][17222:17224] CHIP:TOO:     [4]: {
+          [1689674049.504527][17222:17224] CHIP:TOO:       OperationalStateID: 3
+          [1689674049.504533][17222:17224] CHIP:TOO:      }
+          [1689674049.504605][17222:17224] CHIP:EM: <<< [E:22830i S:37151 M:4250114 (Ack:140781365)] (S) Msg TX to 1:0000000000000001 [BFDE] --- Type 0000:10 (SecureChannel:StandaloneAck)
+          [1689674049.504620][17222:17224] CHIP:IN: (S) Sending msg 4250114 on secure session with LSID: 37151
+      disabled: true
```

Comment:
Please automate this test. Manual tests should be avoided as they make it seem like a test is ready when it's not. Furthermore, this is hard to test by hand because it's complex data parsing.

Reply:
You want me to automate the step-3 right?

Reply:
@tcarmelveilleux Updated the test step, pls review.

Reply:
When I add OperationalStateList attribute and after running Zap_regen_all.py I am getting missing attribute error.
![image](https://github.com/project-chip/connectedhomeip/assets/68609205/16a4b8fd-100e-4821-a6fd-23bc6b3ada4e)

Reply:
above - I don't know where you added that, but it looks like it was in the wrong cluster because those are the features for the basic information cluster. I suspect you may have attempted to automate this step without changing the cluster tag at the top of the test.

Regardless of what happens in this test, if you don't have that attribute defined in the cluster, you're going to fail conformance because it is mandatory. So it should already be there in any app that implements this cluster.

Reply:
the issue is that the cluster config was pointing to "Basic Information" cluster instead of "Oven Cavity Operational State".